fieldservice_agreement/views/agreement_serviceprofile.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="agreement_serviceprofile_notes" model="ir.ui.view">
+        <field name="name">agreement.serviceprofile.notes</field>
+        <field name="model">agreement.serviceprofile</field>
+        <field name="type">form</field>
+        <field name="inherit_id" ref="agreement_serviceprofile.agreement_serviceprofile_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="/form/sheet/group/group[2]" position="after">
+                <group string="Notes">
+                    <field name="notes" nolabel="1" widget="html"/>
+                </group>
+            </xpath>
+            <field name="agreement_id" position="attributes">
+                <attribute name="context">{'default_agreement_id': agreement_id}</attribute>
```

Comment:
@osi-scampbell This doesn't make sense. You are selecting the agreement field and the context should have other fields to help you select the right agreement.

Reply:
When I remove the context from this field and create a serviceprofile from that many2many list on an agreement the agreement_id is not automatically populated. I figured that is the functionality we would want.

Reply:
Which many2many? A service profile belongs to only one agreement:
https://github.com/OCA/contract/blob/12.0/agreement_serviceprofile/models/agreement_serviceprofile.py#L13

Reply:
That is correct I had serviceprofiles as Many2many instead of One2many. I changed all that around. However, when you create a serviceprofile from an agreement, the agreement_id does not autopopulate. That was the intention of that code.